versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     snowflake = {
       source  = "Snowflake-Labs/snowflake"
-      version = "~> 0.69"
+      version = "~> 0.93"
```

I just checked that your changes work fine starting from provider version `0.90`
```suggestion
      version = "~> 0.90"
```